quantum/process_keycode/process_unicode_common.h

```diff
@@ -14,25 +14,49 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef PROCESS_UNICODE_COMMON_H
-#define PROCESS_UNICODE_COMMON_H
+#pragma once
 
 #include "quantum.h"
 
+#if defined(UNICODE_ENABLE) + defined(UNICODEMAP_ENABLE) + defined(UCIS_ENABLE) > 1
+  #error "Cannot enable more than one Unicode method (UNICODE, UNICODEMAP, UCIS) at the same time"
+#endif
+
+// Comma-delimited, ordered list of input modes selected for use (e.g. in cycle)
+// Example: #define UNICODE_SELECTED_MODES UC_WINC, UC_LNX
+#ifndef UNICODE_SELECTED_MODES
+  #define UNICODE_SELECTED_MODES -1
```

Comment:
Wouldn't it be better default behavior to default all modes?

Reply:
No, the default should be that the feature is disabled by default — that way it's opt-in only. This minimizes the chance of anything breaking. For example, if it were enabled by default, some boards that are close to the firmware size limit would break because the feature uses extra memory (namely the `selected` array).